src/app/cic/utils.test.js

```diff
@@ -1,20 +1,24 @@
 const { formatDate } = require("./utils");
 const { expect } = require("chai");
 
-describe("formatDate", () => {
-  it("returns a YYYY-MM-DD date as DD MM YYYY", () => {
-    expect(formatDate("1989-03-31", "YYYY-MM-DD")).to.equal("31 03 1989");
-  });
+describe('formatDate', () => {
+  it('returns a YYYY-MM-DD date with language set to en as DD Month(English) YYYY', () => {
+    expect(formatDate('1989-03-31', "en")).to.equal('31 March 1989');
+  })
 
-  it("returns a YYYY-MM-DD date as DD MM YYYY", () => {
-    expect(formatDate("198903/31", "YYYY-MM-DD")).to.equal("");
-  });
+  it('returns a YYYY-MM-DD date with language set to cy as DD Month(Welsh) YYYY', () => {
+    expect(formatDate('1989-03-31', "cy")).to.equal('31 Mawrth 1989');
+  })
 
-  it("should return an empty string if date is empty string", () => {
-    expect(formatDate("", "YYYY-MM-DD")).to.equal("");
-  });
+  it('returns a YYYYMM/DD date as Invalid date', () => {
+    expect(formatDate('198903/31', "YYYY-MM-DD")).to.equal('Invalid Date');
+  })
```

Comment:
hmm this means that we will be showing "Invalid date" to a user. I think where there is an error we actually probably want to show empty string? Maybe one to check with the BAs?